fineract-provider/src/main/java/org/apache/fineract/cob/service/InlineLoanCOBExecutorServiceImpl.java

```diff
@@ -78,6 +79,7 @@
 @Service
 @Slf4j
 @RequiredArgsConstructor
+@ConditionalOnProperty(value = "fineract.job.loan-cob-enabled", havingValue = "true")
 public class InlineLoanCOBExecutorServiceImpl implements InlineExecutorService<Long> {
```

Comment:
Should we have @Conditional(LoanCOBEnabledCondition.class) instead? I might be wrong but inline COB can be executed only on manager, right?

Reply:
changed